keps/sig-api-machinery/1027-api-unions/README.md

```diff
@@ -102,26 +142,77 @@ proposing the following changes.
 Note that this proposes unions to be "at most one of". Whether exactly one is
 supported or not should be implemented by the validation logic.
 
+### User Stories (Optional)
+
+<!--
+Detail the things that people will be able to do if this KEP is implemented.
+Include as much detail as possible so that people can understand the "how" of
+the system. The goal here is to make this feel real for users without getting
+bogged down.
+
+TODO: ask antoine if these stories are complete/sufficient
+-->
+
+
+#### Story 1
+
+As a CRD owner, I should be able to author my CRD such that the openapi schema
```

Comment:
I'd split into these two stories:
1. CRD Authors don't have to write the validation
2. Type users can commit their intent more easily, especially when a new field they don't know has been introduced.